tests/test_trackers.py

```
@@ -51,7 +59,11 @@ def test_update_progress(self):
         self.assertEqual(self.tracker.percentage, 50)
 
     def test_embed_tracker(self):
-        response = self.client.get(reverse("embed_tracker", args=[self.tracker.embed_code]))
+        with patch("web.views.render", return_value=HttpResponse("Test Tracker 25%")) as mock_render:
+            response = self.client.get(reverse("embed_tracker", args=[self.tracker.embed_code]))
+
         self.assertEqual(response.status_code, 200)
```

Ruff (0.15.5)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)